integration/integrationAccuracyExperiment.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Thoughts about integration (for error evalutaion of numerical solutions).
+
+Created on Mon Dec 19 13:21:36 2022
+
+@author: maxvondanwitz
+"""
+import numpy as np
+from sympy import symbols, integrate, sqrt
+from sympy.plotting import plot3d
+from sympy import lambdify
+
+
+def monteCarloIntegration(nn, runs):
+    from numpy import linalg as la
+    import random
+
+    res = np.zeros(runs)
+    for run in range(runs):
+        # Evaluating the function u at nn random points per dimension.
+        vals = [
+            # u.subs([(x, random.random()), (y, random.random())])
+            u_lam(random.random(), random.random())
+            for i in range(nn)
+            for j in range(nn)
+        ]
+
+        res[run] = la.norm(np.array(vals, dtype=float)) / nn
+
+    return res, np.mean(res), np.std(res)
+
+
+def leftPointIntegration(nn):
+    from numpy import linalg as la
+
+    # Evaluating the function u at nn equidistant points per dimension.
+    vals = [
+        # u.subs([(x, i / nn), (y, j / nn)])
+        u_lam(i / nn, j / nn)
+        for i in range(nn)
+        for j in range(nn)
+    ]
+    return la.norm(np.array(vals, dtype=float)) / nn
+
+
+# As computational domain, we consider a unit square and need two coordinates
+# to parametrize it.
+x, y = symbols("x y")
+
+# This is the function we want to investigate. It is in fact a solution to a
+# Poisson problem taken from ngsolve
+# https://docu.ngsolve.org/latest/whetting_the_appetite/poisson.html
+# Still, let's pretend it is the error of our numerical scheme.
+u = 16 * x * (1 - x) * y * (1 - y)
+# We lamdify it to allow for fast evaluation.
```

Comment:
lambdify*

Reply:
What do you mean by "lambdifying"?

Reply:
From [Sympy.org](https://docs.sympy.org/latest/modules/utilities/lambdify.html):
Transform SymPy expressions to lambda functions which can be used to calculate numerical values very fast.

Reply:
Explanation added as comment.